#2020/01/30 Patch has not yet been accepted upstream
From f4de7a2a21acf1a205b725b4bb5596e1475bad26 Mon Sep 17 00:00:00 2001
From: Eric Timmons <etimmons@mit.edu>
Date: Sun, 8 Dec 2019 14:00:01 -0500
Subject: [PATCH] Fix ARM build using ECL host

ECL appears to not like ~T nested in ~<, even when it's the first argument. So
replace the ~9T that gets triggered during build with spaces.
---
 src/compiler/checkgen.lisp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/compiler/checkgen.lisp b/src/compiler/checkgen.lisp
index a49a0a1fc..026782327 100644
--- src/compiler/checkgen.lisp
+++ src/compiler/checkgen.lisp
@@ -518,14 +518,14 @@ (defun cast-check-uses (cast)
         (cond ((and (ref-p use) (constant-p (ref-leaf use)))
                (warn condition
                      :format-control "~:[This~;~:*~A~] is not a ~
-                       ~<~%~9T~:;~/sb-impl:print-type/:~>~% ~S"
+                       ~<~%        ~:;~/sb-impl:print-type/:~>~% ~S"
                      :format-arguments
                      (list what atype (constant-value (ref-leaf use)))))
               (t
                (warn condition
                      :format-control
                       "~:[Result~;~:*~A~] is a ~/sb-impl:print-type/, ~
-                       ~<~%~9T~:;not a ~/sb-impl:print-type/.~>"
+                       ~<~%        ~:;not a ~/sb-impl:print-type/.~>"
                      :format-arguments (list what dtype atype)))))))
   (values))
 
-- 
2.24.0

